app/src/main/java/com/vultisig/wallet/presenter/keysign/KeysignFlowViewModel.kt

```diff
@@ -419,6 +419,7 @@ internal class KeysignFlowViewModel @Inject constructor(
         try {
             if (nextState == KeysignFlowState.KEYSIGN) {
                 messagesToSign = _keysignPayload!!.getKeysignMessages(_currentVault!!)
+                resetQrAddress()
```

Comment:
Actully it cleans QR address , I renamed it to cleanQrAddress()

It will be used in 
https://github.com/vultisig/vultisig-android/blob/43f11084b0ddb87e70d4671771bb117f11e85cd7/app/src/main/java/com/vultisig/wallet/ui/screens/send/SendScreen.kt#L75-L81